src/80211_if.c

```diff
@@ -1462,6 +1457,11 @@ static void init_hw(struct ieee80211_hw *hw)
 #ifdef CONFIG_PM
 	hw->wiphy->wowlan = &uccp_wowlan_support;
 #endif
+
+	if ((wifi->params.fw_loading == 1) && load_fw(hw)) {
+		UCCP_DEBUG_80211IF("%s-80211IF: FW load failed\n", dev->name);
```

Comment:
For single statement bracket is not needed. Basically we follow linux wireless coding style, we run "checkpatch.pl" on all driver changes to fix styling issues.